images/miq-app/docker-assets/container-scripts/container-deploy-common.sh

```diff
@@ -177,8 +177,8 @@ function write_deployment_info() {
 
 # Prepare appliance initialization environment
 function prepare_init_env() {
-  # Make a copy of CONTAINER_DATA_PERSIST_FILE into PV if not present
-  [ ! -f "${PV_DATA_PERSIST_FILE}" ] && cp -a "${CONTAINER_DATA_PERSIST_FILE}" "${APP_ROOT_PERSISTENT}"
+  # Copy CONTAINER_DATA_PERSIST_FILE into PV ensuring it is kept up to date
+  cp -a "${CONTAINER_DATA_PERSIST_FILE}" "${APP_ROOT_PERSISTENT}"
```

Comment:
@carbonin I have doubts about the original idea of having users modify a file and persist their data. It's not very clean and could lead to issues... 

Regarding this one, the original persist file on PV gets copied only once on normal circumstances, then it is used by the sync-data-pv function always from the copy on PV on a pre-stop event when we shutdown. If a user modifies the persist file in PV , it will always get processed and restored accordingly.

Look here on template : 
https://github.com/ManageIQ/manageiq-pods/blob/master/templates/miq-template.yaml#L181